tests/tests.go

```diff
@@ -286,22 +292,25 @@ func changeReplicas(managerName string, numOfReplica int32) error {
 		}
 
 		return nil
-	}, 30*time.Second, 3*time.Second).ShouldNot(HaveOccurred(), "failed to update number of replicas on manager")
+	}, 30*time.Second, 3*time.Second).ShouldNot(HaveOccurred(), "failed to update number of replicas on deployment:\n%v", string(indentedDeployment))
 
 	By(fmt.Sprintf("Waiting for expected ready pods to be %d", numOfReplica))
 	Eventually(func() bool {
 		managerDeployment, err := testClient.KubeClient.AppsV1().Deployments(managerNamespace).Get(context.TODO(), managerName, metav1.GetOptions{})
 		if err != nil {
 			return false
 		}
+		indentedDeployment, err = json.MarshalIndent(managerDeployment, "", "\t")
+		if err != nil {
+			return false
+		}
 
 		if managerDeployment.Status.ReadyReplicas != numOfReplica {
 			return false
 		}
 
 		return true
-
-	}, 2*time.Minute, 3*time.Second).Should(BeTrue(), "failed to change kubemacpool deployment number of replicas")
+	}, 2*time.Minute, 3*time.Second).Should(BeTrue(), fmt.Sprintf("failed to change kubemacpool deployment number of replicas.\n deployment:\n%v", string(indentedDeployment)))
```

Comment:
To/Should cláusula at Ginkgo works like a Sprintf you can just pass the fmt string and the expansion values.

Reply:
cool! thanks. Done